Volume/Views/MainView/MagazineDetail.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  MagazineDetail.swift
+//  Volume
+//
+//  Created by Justin Ngai on 9/3/2022.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct MagazineDetail: View {
+    var body: some View {
+        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
```

Comment:
what are these comments?

Reply:
It's the default text new SwiftUI view. Will replace when I implement the MagazineDetail view